### ISSUE TITLE
SyntaxError: Unexpected token '>'

### ISSUE THREAD

**Issue by OKNoah** [leandrowd/react-responsive-carousel#52]
If react-responsive-carousel is imported in my project, the app crashes with the error `SyntaxError: Unexpected token '>'`.

```
import React, {Component} from 'react'
import Carousel from 'react-responsive-carousel'

export default class Portfolion extends Component {
  render () {
    return (
      <div className={classes.portfolio}>
        <h1>Portfolio</h1>
        <h2>Slipstream</h2>
        <Carousel.Carousel
          showStatus={false}
        >
          <div>
            <img src="sdjfakl" alt="" />
          </div>
        </Carousel.Carousel>
      </div>
    )
  }
}
```

Using

```
"react": "^15.1.0",
"react-responsive-carousel": "^3.0.13",
"node": "5.10.0"
```

**Comment by leandrowd** [comment]
Hi @OKNoah, I tested it here and worked fine. 

Check the demo: https://leandrowd.github.io/demo-react-responsive-carousel-es6/
Source available at: https://github.com/leandrowd/demo-react-responsive-carousel-es6/blob/master/index.js

Could you check if following this example it works? Or provide more details so I can investigate. 

Thanks!

**Comment by OKNoah** [comment]
@leandrowd That page also shows the error in my browser. I'm in Safari 9. Safari Technology Preview does not show the error.

**Comment by leandrowd** [comment]
Ahh cool, I tested on Safari and found the problem. The bundle was not transpiling to full es5, there were still some features like arrow functions in the lib, which works on Chrome but not on Safari yet. I fixed that and published 3.0.15. 

Thanks for raising the issue.
